reflex-dom-contrib.cabal

```diff
@@ -46,19 +46,19 @@ library
     Reflex.Dom.Contrib.Widgets.Svg
     Reflex.Dom.Contrib.Widgets.ScriptDependent
 
-  if impl(ghcjs)
-   exposed-modules:
-     Reflex.Dom.Contrib.Geoposition
+--  if impl(ghcjs)
```

Comment:
Oh, I'm not sure.  I don't think I touched anything regarding geoposition, but perhaps this was broken previously.